source/faq.txt

```diff
@@ -20,20 +20,20 @@ Support.
 Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
 ---------------------------------------------------------------------------------------
 
-You can perform reads during synchronization if ``canCommit`` is ``true``. 
-However, the data that you read is 
-:term:`eventually consistent <eventual consistency>`, meaning that you might not 
-always read the latest writes.
+You can perform reads during synchronization at any time. However, the data 
+that you read is :term:`eventually consistent <eventual consistency>`, meaning 
+that you might not always read the latest writes. For up-to-date reads, wait 
+for the migration to commit.
 
 During an ongoing sync, you can write to any non-synced namespaces in the 
```

Comment:
If you're using namespace filtering some namespaces may not be synced and can be considered non-synced namespaces

Reply:
Gotcha, thanks. I'm not following what it means to have a non-synced namespace in the destination cluster - does that mean the customer can issue random writes to the destination on namespaces not included in the mongosync, as long as it happens after mongosync begins?

Reply:
There are two cases:
- A GA sync. If there is any data on the destination at the start of the sync, the migration will error. If clients write to a non-included namespace while the migration is going on, this should not cause problems with mongosync. They'd have to be really careful to not write to a synced namespace or they will cause data inconsistency. It does make it harder to verify, for example, when 1.9 goes out if they want to use the embedded verifier, if they write to a non-included namespace without specifying namespace filtering, then the embedded verifier will detect the inconsistency (being the writes to the non-synced namespaces)
- A beta sync with ignorePreExistingNamespaces. In this case mongosync will ignore existing data but error out on any namespace collisions. There is no verification for this case.

I think it would make sense to remove that sentence, because it might just cause confusion.

Reply:
I'm good to remove that sentence, thanks!